oauthenticator/oauth2.py

```diff
@@ -268,7 +302,7 @@ def login_url(self, base_url):
 
     login_handler = "Specify login handler class in subclass"
```

Comment:
I saw these changes removed custom login handlers for some classes. In that case, they will default to the `login_handler` value defined here. Should this instead fall back to `OAuthLoginHandler` if the subclass doesn't define a custom Login handler?

Reply:
> Should this instead fall back to OAuthLoginHandler if the subclass doesn't define a custom Login handler?

Yes, absolutely!